pyramid/scripts/pshell.py

```diff
@@ -77,6 +96,8 @@ def pshell_file_config(self, filename):
         for k, v in items:
             if k == 'setup':
                 self.setup = v
+            elif k == 'default_shell':
```

Comment:
I started with that and changed to `default_shell` but I can change it back. I don't really care. Naming is hard. Next someone is going to ask to load the option from an environment variable anyway.